components/cam/src/chemistry/modal_aero/aero_model.F90

```diff
@@ -1515,20 +1525,32 @@ subroutine aero_model_wetdep(dt, dlf, dlf2, cmfmc2, state,                    &!
     ncol  = state%ncol
 
     call physics_ptend_init(ptend, state%psetcols, 'aero_model_wetdep_ma', lq=wetdep_lq)
-    
+
     ! Do calculations of mode radius and water uptake if:
     ! 1) modal aerosols are affecting the climate, or
     ! 2) prognostic modal aerosols are enabled
-    
-    call t_startf('calcsize')
-    ! for prognostic modal aerosols the transfer of mass between aitken and accumulation
-    ! modes is done in conjunction with the dry radius calculation
-    call modal_aero_calcsize_sub(state, ptend, dt, pbuf)
-    call t_stopf('calcsize')
-
-    call t_startf('wateruptake')
-    call modal_aero_wateruptake_dr(state, pbuf)
-    call t_stopf('wateruptake')
+    ! If not using prognostic aerosol call the diagnostic version
```

Comment:
It actually switches to the BAM version of aero_model_wetdep, but in either case I think you're right that's it's safe to remove this. I originally put it there because I wasn't 100% sure how things were behaving and thought it was safer this way, but I'm more confident that removing this is safe. So I'll remove it.